.scripts/automation_generate.sh

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-hlc-code-gen-for-pipeline --inputJsonPath=$1 --outputJsonPath=$2 --use=@autorest/typescript@6.0.0-alpha.16.20220105.1
+code-gen-pipeline --inputJsonPath=$1 --outputJsonPath=$2 --use=@autorest/typescript@6.0.0-alpha.16.20220105.1
```

Comment:
Can we wait to use tomorrow's nightly build as there's some bugfixes I would like the automation pipeline to take ?